examples/python/gRPC/images/gRPC_fb_addBoundingBox.py

```diff
@@ -7,8 +7,8 @@
 
 import flatbuffers
 import numpy as np
-from fb import BoundingBox2DLabeledWithCategory, BoundingBoxes2DLabeledStamped, Image
```

Comment:
I guess the deleted `BoundingBox2DLabeledWithCategory` is still needed and should be added again

Reply:
Nope, not bounding box import needed here, it's handled by the `util_fb`